addons/mail/static/src/core/web/thread_model_patch.js

```diff
@@ -38,14 +38,16 @@ patch(Thread.prototype, {
         const data = await this.store.env.services.orm.call(this.model, "message_get_followers", [
             [this.id],
             this.followers.at(-1).id,
+            false,
+            this.followers.length,
         ]);
         this.store.insert(data);
     },
     async loadMoreRecipients() {
         const data = await this.store.env.services.orm.call(
             this.model,
             "message_get_followers",
-            [[this.id], this.recipients.at(-1).id],
+            [[this.id], this.recipients.at(-1).id, false, this.recipients.length],
             { filter_recipients: true }
         );
```

Comment:
The ordering by id had the benefit of ensuring that newer followers are necessarily at the end, which makes "load more" works even when list is memoized for a long time.

With the sort alphabetical, this is no longer true, and this can lead to follower that cannot have their data fetched. For example, if we fetch 20 followers whose name start with `A` to `D`, and then I follow the task ("Alex"), you won't be able to fetch my follower because load more works only after `D`.

One solution is to scrap the list of followers each time you close the popover, and we need to necessarily fetch list again. This is less efficient, but the sorting by name requires that. If we force us to be efficient with fetch, the code becomes far too complex to the point so that's why start fetch from start again makes more sense.

Reply:
Now we're reinitiating the followers list each time we open the popover, as you suggested